app/src/main/java/com/example/plantsapp/model/ListOfPlants.kt

```diff
@@ -0,0 +1,12 @@
+package com.example.plantsapp.model
+
+const val SEVEN_DAYS: Int = 7
+const val PLANT_PICTURE = "https://www.vippng.com/png/detail/41-414674_house-plant-png.png"
+
+val listOfPlants = listOf(
+    Plant("Bob", "Succulent", PLANT_PICTURE, SEVEN_DAYS),
+    Plant("Marley", "Spath plant", PLANT_PICTURE, SEVEN_DAYS),
+    Plant("John", "Cacti", PLANT_PICTURE, SEVEN_DAYS),
+    Plant("Casey", "Tillandsia", PLANT_PICTURE, SEVEN_DAYS),
+    Plant("Robert", "Succulent", PLANT_PICTURE, SEVEN_DAYS)
+)
```

Comment:
Just move it to PlantsViewModel - because these hardcoded data is only needed inside of this viewmodel and will be removed in future

And make them private